src/adapters/Base/index.ts

```diff
@@ -0,0 +1,30 @@
+import { AxiosRequestConfig } from 'axios';
+
+import { JSONObject, keysToSnakeCase } from 'helpers/json';
+import requestManager from 'lib/requestManager';
+
+const BaseAdapter = () => {
+  const get = (path: string, params?: JSONObject) => {
+    const requestOptions: AxiosRequestConfig = {};
+    if (params) {
+      requestOptions.params = keysToSnakeCase(params);
```

Comment:
Recommend to use the `Object.assign` instead of mapping it directly to an object

Reply:
@nvminhtue the `requestOptions.params` is null by default so we cannot use the Object.assign for the `null` property, so I use that method instead 🙏 
![image](https://github.com/manh-t/react-survey/assets/60863885/a8d37893-e2a4-4760-aba6-f79bf4316812)